website/docs/guides/1.5.0-upgrade-guide.html.markdown

```diff
@@ -0,0 +1,33 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas Provider 1.5.0: Upgrade and Information Guide"
+sidebar_current: "docs-mongodbatlas-guides-150-upgrade-guide"
+description: |-
+MongoDB Atlas Provider 1.5.0: Upgrade and Information Guide
+---
+
+# MongoDB Atlas Provider 1.5.0: Upgrade and Information Guide
+
+The Terraform MongoDB Atlas Provider version 1.5.0 has a number of new and exciting features and changes.
+
+New Features:
+* You can now manage AWS Serverless Private Endpoints [`mongodbatlas_serverless_private_endpoints`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/serverless_private_endpoints)
+* You can now manage AtlasGov Projects [`mongodbatlas_mongodbatlas_project`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/project)
```

Comment:
Can we also add a line to enable the provider for use with AtlasGov and point to the getting started guide which will have a new section?   The URL should be https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs#configure-mongodb-atlas-for-government

Reply:
Updated, let me know if all looks good